internal/grpc/client/manager.go

```diff
@@ -73,13 +75,21 @@ func (m *Manager) StreamTaskOutput(ctx context.Context, taskID string) error {
 		return fmt.Errorf("error starting output stream: %w", err)
 	}
 
-	// TODO handle output stream once implemented
-	_, err = stream.Recv()
-	if err != nil {
-		return fmt.Errorf("error receiving from stream: %w", err)
+	for {
+		resp, err := stream.Recv()
+		if err == io.EOF {
```

Comment:
Suggestion: prefer errors.Is
```suggestion
		if errors.Is(err, io.EOF) {
```